codespell_lib/data/dictionary.txt

```diff
@@ -33568,30 +33599,42 @@ queiried->queried
 queisce->quiesce
 queriable->queryable
 quering->querying
+querried->queried
 querries->queries
 querry->query, quarry,
+querrying->querying
 queryies->queries
 queryinterace->queryinterface
 querys->queries
 quesant->croissant
 quesants->croissants
 queset->quest
+quesetion->question
+quesetions->questions
 quesets->quests
+quesion->question
+quesions->questions
 quesiton->question
 quesitonable->questionable
 quesitons->questions
 quesr->quest
+quesrion->question
+quesrions->questions
```

Comment:
The -ed form could be added here as well.

Reply:
Perhaps -ing too?

Reply:
And questionnaire(s)?

Reply:
Done